integrations/clickup/client.py

```diff
@@ -0,0 +1,192 @@
+from typing import Any, AsyncGenerator, Optional
+from enum import StrEnum
+import httpx
+from httpx import Timeout
+from loguru import logger
+from port_ocean.utils import http_async_client
+from port_ocean.context.event import event
+
+
+class CacheKeys(StrEnum):
+    TEAMS = "TEAMS"
+    SPACES = "SPACES"
+    TASKS = "TASKS"
+
+
+class ObjectKind(StrEnum):
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
```

Comment:
why this is in the client file, while it is never used here? the `ObjectKind` resource should only be known by the `resync` logic and not by the client

Reply:
Bad mistake. Thanks for the correction